html/index.html

```diff
@@ -0,0 +1,713 @@
+          </div>
+          <div class="circle">
+            <svg
+              class="arrow-icon --arrow-hover"
+              width="9"
+              height="14"
+              fill="none"
+              stroke="#64BE97"
+            >
+              <use xlink:href="/images/sprite.svg#arrow-right"></use>
+            </svg>
+          </div>
+        </div>
+      </section>
+
+      <section class="customers-block">
+        <div class="customers-block____images">
+          <div class="customers-block____images-square"></div>
+          <img
+            class="customers-block____images-img"
+            width="270.03"
+            height="270.03"
+            src="/images/customers-block/customers-photo.png"
+            alt="customers-photo"
+          />
+        </div>
+
+        <div class="customers-block____container">
+          <div class="customers-block____container-background">
+            <img
+              class="customers-block____container-background-one"
+              width="796"
+              src="/images/customers-block/background_1.png"
+              alt="background_1"
+            />
+            <img
+              class="customers-block____container-background-two"
+              width="515"
+              src="/images/customers-block/background_2.png"
+              alt="background_2"
+            />
+          </div>
+
+          <div class="customers-block____container-info">
+            <div class="customers-block____container-info-title">
+              <h2 class="block-title">Our great customers</h2>
+              <div class="title-underline"></div>
+            </div>
+
+            <img src="/images/customers-block/quotes.png" alt="quotes" />
+            <div>
+              <p>
+                Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium
+                doloremque
+              </p>
+              <p>
+                Laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+                architecto beatae vitae dicta sunt explicabo.
+              </p>
+              <p>Stephanie Magion</p>
+            </div>
+
+            <div class="customers-block____container-info-circles">
+              <div class="customers-block____container-info-circles-circle"></div>
+              <div class="customers-block____container-info-circles-circle"></div>
+              <div class="customers-block____container-info-circles-circle blue"></div>
+              <div class="customers-block____container-info-circles-circle"></div>
+              <div class="customers-block____container-info-circles-circle"></div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="started-block">
+        <!-- <img
+          src="/images/started-block/started-background.png"
+          alt="started-background"
+          width="1440"
+          height="406"
+        /> -->
```

Comment:
Delete unnecessary code. Don't comment it. It's bad practice

Reply:
Done.